src/pyelliptic/eccblindChain.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+"""
+ECC blind signature functionality based on "An Efficient Blind Signature Scheme
+Based on the Elliptic CurveDiscrete Logarithm Problem" by Morteza Nikooghadama
+<mnikooghadam@sbu.ac.ir> and Ali Zakerolhosseini <a-zaker@sbu.ac.ir>,
+http://www.isecure-journal.com/article_39171_47f9ec605dd3918c2793565ec21fcd7a.pdf
+"""
+
+# variable names are based on the math in the paper, so they don't conform
+# to PEP8
+# pylint: disable=invalid-name
+
+import sys
+import msgpack
+import hash as Hash
+sys.path.insert(1, '/home/cis/ektarepo/PyBitmessage/src/pyelliptic')
+from eccblind import ECCBlind
+
+
+def encode_datetime(obj):
+    """
+    Method to format time
+    """
+    return {'Time': int(obj.strftime("%s"))}
+
+
+class ECCBlindChain(object):  # pylint: disable=too-many-instance-attributes
+    """
+    # Class for ECC Blind Chain signature functionality
+    """
+    signature = None
+
+    def ec_serialize(self, msg):
+        """
+        Serialize the data using msgpack
+        """
+        msg = msgpack.packb(msg, default=encode_datetime, use_bin_type=True)
+        return msg
+
+    def ec_deSerialize(self, msg):
+        """
+        Deserialize the data using msgpack
+        """
+        msg = msgpack.unpackb(msg)
```

Comment:
`self.signature = msgpack.unpackb(msg)`
Also use a different variable name, not `msg` but e.g. `chain`

Reply:
and finish with `return self`